hera_cal/abscal_funcs.py

```diff
@@ -311,9 +325,24 @@ def phs_logcal(model, data, wgts=None, verbose=True):
     fill_dict_nans(ydata, wgts=wgts, nan_fill=0.0, inf_fill=0.0)
 
     # setup linsolve equations
-    eqns = odict([(k, "phi_{}_{} - phi_{}_{}".format(k[0], k[-1][0], k[1], k[-1][1])) for i, k in enumerate(keys)])
+    eqns = []
+    for i, k in enumerate(keys):
+        ant1 = "phi_{}_{}".format(k[0], k[2][0])
+        ant2 = "phi_{}_{}".format(k[1], k[2][1])
+        if refant == k[0]:
+            ant1 += "*a1"
+        elif refant == k[1]:
+            ant2 += "*a1"
+        eqns.append((k, "{} - {}".format(ant1, ant2)))
+    eqns = odict(eqns)
     ls_design_matrix = odict()
 
+    # set reference antenna phase to zero
+    if refant is None:
+        refant = keys[0][0]
+    assert np.array(map(lambda k: refant in k, keys)).any(), "refant {} not found in data and model".format(refant)
+    ls_design_matrix['a1'] = 0.0
```

Comment:
I'm confused by this. Doesn't this just make the coefficient of the reference anntenna's phase 0? This doesn't force it to be zero... it means that it could be anything and still satisfy the equations, no?